demos/src/main.scss

```diff
@@ -6,3 +6,20 @@ body {
 	font-family: 'MetricWeb';
 	background: #fff1e0;
 }
+
+.demo-sticky {
+	height: 2999px;
```

Comment:
The registry looks like it switches off scrolling... could this be used as a wrapper, instead?